src/main/java/com/redhat/cloud/policies/app/PoliciesHistoryCleaner.java

```diff
@@ -0,0 +1,47 @@
+package com.redhat.cloud.policies.app;
+
+import io.quarkus.scheduler.Scheduled;
+import org.eclipse.microprofile.config.ConfigProvider;
+import org.hibernate.Session;
+import org.jboss.logging.Logger;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import javax.transaction.Transactional;
+import java.time.Duration;
+import java.time.Instant;
+
+import static java.util.concurrent.TimeUnit.MINUTES;
+
+@Singleton
+public class PoliciesHistoryCleaner {
+
+    public static final String POLICIES_HISTORY_CLEANER_DELETE_AFTER_CONF_KEY = "policies-history.cleaner.delete-after";
+
+    private static final Logger LOGGER = Logger.getLogger(PoliciesHistoryCleaner.class);
+    private static final Duration DEFAULT_DELETE_DELAY = Duration.ofDays(14L);
+
+    @Inject
+    Session session;
+
+    /**
+     * The policies UI shows a retention time for policies history entries.
+     * This scheduled job deletes from the database the history entries which are older than that retention time.
+     */
+    /*
+     * TODO The scheduling is delayed to prevent an unwanted execution during tests. Remove the delay and set the period
+     * to `disabled` after the Quarkus 2 bump. See https://quarkus.io/guides/scheduler-reference for more details.
+     */
```

Comment:
This is not a 2.0 feature, at least since 1.7: https://quarkus.io/version/1.7/guides/scheduler-reference#quarkus-scheduler_quarkus.scheduler.enabled

You could use:
`%test.quarkus.scheduler.enabled=false`

Reply:
I know the conf property to disable the scheduler globally but 2.0 will introduce the possibility to disable each scheduled job individually. Tests will fail in this project if `%test.quarkus.scheduler.enabled=false`, I tried it already.

Reply:
These are new in Quarkus 2:

> If you wish to disable a specific scheduled method, you can set its cron expression to "off" or "disabled".

> Intervals can be disabled by setting their value to "off" or "disabled". So for example a Property Expression with the default value "off" can be used to disable the trigger if its Config Property has not been set.